src/View/Form/ArrayContext.php

```diff
@@ -121,12 +122,34 @@ public function primaryKey(): array
         return [];
     }
 
+    /**
+     * Get the fields used in the context as a primary key.
+     *
+     * @return string[]
+     */
+    public function getPrimaryKey(): array
```

Comment:
This looks like the same implementation as `primaryKey()` has. Why duplicate the code?

Reply:
Just an accident.  I left it in fixing a warning.